.github/workflows/version-bump.yml

```diff
@@ -1,54 +1,37 @@
 # **what?**
-# This workflow will take a version number and a dry run flag. With that
+# This workflow will take the new version number to bump to. With that
 # it will run versionbump to update the version number everywhere in the
-# code base and then generate an update Docker requirements file. If this
-# is a dry run, a draft PR will open with the changes. If this isn't a dry
-# run, the changes will be committed to the branch this is run on.
+# code base and then run changie to create the corresponding changelog.
+# A PR will be created with the changes that can be reviewed before committing.
 
 # **why?**
 # This is to aid in releasing dbt and making sure we have updated
-# the versions and Docker requirements in all places.
+# the version in all places and generated the changelog.
 
 # **when?**
-# This is triggered either manually OR
-# from the repository_dispatch event "version-bump" which is sent from
-# the dbt-release repo Action
+# This is triggered manually
 
 name: Version Bump
 
 on:
   workflow_dispatch:
     inputs:
       version_number:
-       description: 'The version number to bump to'
+       description: 'The version number to bump to (ex. 1.2.0, 1.3.0b1)'
        required: true
-      is_dry_run:
-       description: 'Creates a draft PR to allow testing instead of committing to a branch'
-       required: true
-       default: 'true'
-  repository_dispatch:
-    types: [version-bump]
 
 jobs:
   bump:
     runs-on: ubuntu-latest
     steps:
+      - name: "[DEBUG] Print Variables"
```

Comment:
Is this still needed?

Reply:
Yes, I see how this could look like a remnant of testing so maybe we rename this in `main` but it just allows us to easily see what the input is since GHA doesn't show you that automatically

Reply:
@ChenyuLInx this is in the GHA standards [README](https://github.com/dbt-labs/dbt-core/tree/main/.github).  I am open to renaming this step, but the step itself is valuable.

Reply:
Makes total sense!! I think I left that comment with the first impression of this is something being used to debug during developing